index.js

```diff
@@ -37,12 +37,14 @@ module.exports = function headless(options, startnum, callback) {
   }
 
   findFreeServernum(startnum, function(servernum) {
-    if (!options) {
-      var childProcess = spawn('Xvfb', [':' + servernum]);
-    }
-    else {
-      var childProcess = spawn('Xvfb', [':' + servernum , '-screen' , '0' , options.display.width + 'x' + options.display.height + 'x' + (options.display.depth || 16) ]);
+    options = options || {}
+
+    var args = options.args || []
```

Comment:
we can move this higher up - no need to do it in the findFreeServernum-method, right?